### ISSUE TITLE
Vagrant up error: not a user me@mailinabox

### ISSUE THREAD

**Issue by cromulus** [mail-in-a-box/mailinabox#1026]
Can't seem to boot the vagrant dev environment:

```
Invalid email address.
==> default: FAILED: tools/mail.py user make-admin me@mailinabox
==> default: -----------------------------------------
==> default: That's not a user (me@mailinabox).
==> default: -----------------------------------------
```

Seems like the problem is Vagrantfile:12: `config.vm.hostname = "mailinabox"` 
might need fully qualified domain name or some reasonable equivalent.

**Comment by JoshData** [comment]
Probably broken by #1005. Paging @wsteitz!

**Comment by wsteitz** [comment]
I actually never managed to get the miab vagrant dev box to work. Removing the justtesting domain in #1005 only fixed that problem. I confirm that changing the hostname to a fully qualified one (e.g. mailinabox.lan) fixes it.

**Comment by guyzmo** [comment]
I confirm that too. I hit the issue, but solved it "mechanically" without giving it too much thought (maybe because I know MIAB well enough now ☺). I changed it to use `mailinabox.tld`.

Also I had to add `mailinabox.tld` pointing to the VM's IP in my `/etc/hosts`, so when you open the IP in your browser it does not refresh to an unknown webpage.

Maybe it'd be an idea to have the Vagrantfile configure MIAB to use the IP address of the VM instead of bogus domain name, so that no hack is necessary for development purposes?
